oap-server/server-starter/src/main/resources/ui-initialized-templates/menu.yaml

```diff
@@ -0,0 +1,147 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# menus: the menus of the left navigation bar
+#   - name: the name of the menu item
+#     icon: icon name
+#     path: link path
+#     dashboard: related dashboard name, the menu item should be activated when the dashboard value is empty or dashboard related scope has services
+#     menus: sub menus of the current item
+
+menus:
+  - name: General Service
+    icon: chart
+    menus:
+      - name: Services
+        path: /general
+        dashboard: General-Root
```

Comment:
Are you going to use the dashboard name? So, the dashboard ID would be irrelevant

Reply:
It's dashboard ID, I needs to using the dashboard ID to query the dashboard configuration.

Reply:
If that is the ID, let's name it as ID.

Reply:
This doc should be updated as well, https://skywalking.apache.org/docs/main/next/en/ui/readme/. Add menu change introduction.